.travis.yml

```diff
@@ -1,29 +1,25 @@
+xcode_workspace: Commandant.xcworkspace
+git:
+  submodules: false
+before_install:
+  - git submodule update --init --recursive
 script: placeholder # workaround for https://github.com/travis-ci/travis-ci/issues/4681
 matrix:
   exclude:
     - script: placeholder # workaround for https://github.com/travis-ci/travis-ci/issues/4681
   include:
-    - script: script/cibuild
+    - script: set -o pipefail && xcodebuild -workspace "$TRAVIS_XCODE_WORKSPACE" -scheme "$TRAVIS_XCODE_SCHEME" test | xcpretty
+      xcode_scheme: Commandant
       os: osx
       osx_image: xcode7.3
       language: objective-c
       env: JOB=Xcode7.3
-      xcode_workspace: Commandant.xcworkspace
+    - script: set -o pipefail && xcodebuild -workspace "$TRAVIS_XCODE_WORKSPACE" -scheme "$TRAVIS_XCODE_SCHEME" test | xcpretty
       xcode_scheme: Commandant
-    - script: swift build
       os: osx
-      osx_image: xcode7.3
+      osx_image: xcode8
       language: objective-c
-      env: JOB=SPM
-      install:
-        - eval "$(curl -sL https://gist.githubusercontent.com/kylef/5c0475ff02b7c7671d2a/raw/02090c7ede5a637b76e6df1710e83cd0bbe7dcdf/swiftenv-install.sh)"
-    - script: swift build
-      env: JOB=Linux
-      sudo: required
-      dist: trusty
-      language: generic
-      install:
-        - eval "$(curl -sL https://gist.githubusercontent.com/kylef/5c0475ff02b7c7671d2a/raw/02090c7ede5a637b76e6df1710e83cd0bbe7dcdf/swiftenv-install.sh)"
```

Comment:
I tried to update to a recent snapshot as of https://github.com/jpsim/Commandant/tree/jp-05-31-snapshot, but that requires breaking changes and I gave it up. So this should be my fault at that time.